yang/ietf/ietf-interfaces@2018-02-20.yang

```diff
@@ -0,0 +1,1123 @@
+module ietf-interfaces {
```

Comment:
We need to decide whether we should augment this module with FRR specific information, or vice versa. In any case this module duplicates the existing `frr-interface` module and that needs to be resolved.